### PR TITLE
First RVV backend version

### PR DESCRIPTION
**Pull request by adcastel**
I have included madds, loads, stores, bcast and fmul operations for RVV so it can be the base for the RVV-EXO development. I will add the test with the expected output ASAP.

### CODE REVIEW

File: .idea/exo.iml [hunk 1]
```diff
@@ -5,8 +5,9 @@
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/tests" isTestSource="true" />
       <excludeFolder url="file://$MODULE_DIR$/dependencies/chipyard" />
+      <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.11 (exo)" jdkType="Python SDK" />
```
Comment (yamaguchi1024) [inline .idea/exo.iml:10]: @adcastel What are these changes for?

**Comment by adcastel** [comment]
I think that file comes from the pycharm environment. I will check it.Adrián Castelló El 14 jun 2023, a las 4:20, Yuka Ikarashi ***@***.***> escribió:﻿
@yamaguchi1024 commented on this pull request.



In .idea/exo.iml:
>      </content>
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.11 (exo)" jdkType="Python SDK" />

@adcastel What are these changes for?

File: src/exo/query_asts.py [hunk 2]
```diff
@@ -97,6 +97,9 @@ def __init__(self):
 class R(Type):
     pass
 
+@_dataclass
```
Comment (yamaguchi1024) [inline src/exo/query_asts.py:100]: @adcastel This is a duplication of the code below

Comment (adcastel) [reply src/exo/query_asts.py:100]: Fixed

**Review by yamaguchi1024 — APPROVED** [review]

**Pull request merged**